app/src/main/java/swen_anigans/mathematicfanatic/ClassroomActivity.java

```diff
@@ -168,21 +150,18 @@ public void goToLocker(View view){
     }
 
     public void checkAnswer(View view){
-        EditText answerInput = (EditText) findViewById(R.id.editClassroomAnswer);
-        String answer = answerInput.getText().toString();
-        CharSequence text;
-        if (!answer.isEmpty()) {
-            int quizAnswer = Integer.parseInt(answer);
-            answers.set(pageNumber-1, quizAnswer);
-            boolean correct;
-            if(quizAnswer == expectedAnswers.get(pageNumber - 1)){
-                text = "Correct!";
+        TextView answerInput = (TextView) findViewById(R.id.editClassroomAnswer);
+        String answerText = answerInput.getText().toString();
+        String text;
+        if(!answerText.isEmpty()){
```

Comment:
Sure. it was just that we were basically doing the same exact thing as saveAnswer (find view, check to see if the answer was empty, then setting it to either what we have or 0)

Was thinking about code reuse, but that's not really a big deal for this. It is fine this way!